### ISSUE TITLE
Super Terceira Idade

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments#153]
Post de referência: https://www.vish.blog.br/imagens/super-terceira-idade/

**Comment by brunoaugusto** [comment]
A cara do Hulk "ODEIO MÉDICO!" :rofl:

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/super-terceira-idade/#comment_1128945682>):

> A cara do Hulk "ODEIO MÉDICO!" 🤣

Isso se é que aquela placa diz "médico" ou algo do tipo

**Comment by angellisy** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/super-terceira-idade/#comment_1129107232>):

> Está escrito "Oculista". Oftalmologista, que seria o mais apropriado, é uma palavra bem maior em russo

Você fala russo? :open\_mouth:

**Comment by najarin-the-loremaster** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/super-terceira-idade/#comment_1129141043>):

> Você fala russo? 😮

Não, sou fluente em Google Translate ou, mais especificamente nesse caso, [Yandex Translate](<https://translate.yandex.com/ocr>) que traduziu o texto a partir de um pedacinho da imagem
